cmd/root.go

```diff
@@ -0,0 +1,94 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	cloudclient "github.com/calyptia/api/client"
+	cnfg "github.com/calyptia/cli/cmd/config"
+	"github.com/calyptia/cli/cmd/top"
+	"github.com/calyptia/cli/cmd/version"
+	cfg "github.com/calyptia/cli/config"
+	"github.com/calyptia/cli/localdata"
+)
+
+func NewRootCmd(ctx context.Context) *cobra.Command {
+	client := &cloudclient.Client{
+		Client: http.DefaultClient,
+	}
+
+	localData := localdata.New(cnfg.ServiceName, cnfg.BackUpFolder)
+	config := &cfg.Config{
+		Ctx:       ctx,
+		Cloud:     client,
+		LocalData: localData,
+	}
+
+	token, err := localData.Get(cnfg.KeyToken)
+	if err != nil && errors.Is(err, localdata.ErrNotFound) {
+		cobra.CheckErr(fmt.Errorf("could not retrieve your stored token: %w", err))
+	}
+
+	cloudURLStr, err := localData.Get(cnfg.KeyBaseURL)
+	if err != nil && errors.Is(err, localdata.ErrNotFound) { // if BaseURL not found in local storage revert to default value without warning or error
```

Comment:
@gabrielbussolo I removed the error message from here and I have it take the default value if none is configured. For the better UX

Reply:
Please don't ignore the error.

Reply:
Should we do if !is.Error...?
At the moment this is blocking CI to finish tests.